### PR TITLE
add menu swap for buy-plank at sawmill operator

### PR DESCRIPTION
**Pull request by josiahlund**
First pull request. Hope this helps. I'd love any feedback.

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java [hunk 1]
```diff
@@ -242,6 +242,16 @@ default boolean swapAbyssTeleport()
 		return true;
 	}
 
+	@ConfigItem(
+			keyName = "swapPlank",
```
Comment (TheStonedTurtle) [inline runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java]: fix indent level

Comment (lund8619) [reply runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java]: To do that, do I need to cancel the pull request, make the fix, then resubmit?

Comment (Spedwards) [reply runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java]: No, just made the changes, then commit and push.

Comment (TheStonedTurtle) [reply runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java]: probably better to just fix the original commit with a reset & force push

Comment (lund8619) [reply runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java]: I think I've taken care of it now.

**Comment by lund8619** [comment]
> Looks good.
Thanks. And to those poor ironmen out there... I am one haha

**Comment by Digbigpig** [comment]
Duplicate of #6914

**Comment by lund8619** [comment]
For issue #1930

**Comment by melkypie** [comment]
This was added in menu swapper extended in the plugin hub

**Pull request closed without merging**